src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const convertsObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `convertsObject`. It decreases code readability. Consider using a more descriptive name that reflects the purpose of the variable, such as `styleObject` or `cssProperties`.